### ISSUE TITLE
Error on installing mods on linux when encoding is not en_US

### ISSUE THREAD

**Issue by michal2229** [KSP-CKAN/CKAN#3123]
Background
----------
* Operating System: Ubuntu 20.04, Polish locale
* CKAN Version: v1.28
* KSP Version: 1.10

**Have you made any manual changes to your GameData folder (i.e., not via CKAN)?**
Nope, removed previously installed mods.

Problem
-------
**Describe the bug**
CKAN launches correctly. When attempting to install a mod, following error message appears and the application is terminated:
```
ERROR CKAN.ErrorDialog (null) - Unhandled exception:
System.NotSupportedException: Encoding 852 data could not be found. Make sure you have correct international codeset assembly installed and enabled.
  at System.Text.Encoding.GetEncoding (System.Int32 codepage) [0x0024d] in <12b418a7818c4ca0893feeaaf67f1e7f>:0 
  at ICSharpCode.SharpZipLib.Zip.ZipConstants.ConvertToString (System.Byte[] data, System.Int32 count) [0x00017] in <29fa7683355b471a9cc202367184a830>:0 
  at ICSharpCode.SharpZipLib.Zip.ZipConstants.ConvertToStringExt (System.Int32 flags, System.Byte[] data, System.Int32 count) [0x00032] in <29fa7683355b471a9cc202367184a830>:0 
  at ICSharpCode.SharpZipLib.Zip.ZipFile.ReadEntries () [0x00312] in <29fa7683355b471a9cc202367184a830>:0 
  at ICSharpCode.SharpZipLib.Zip.ZipFile..ctor (System.String name) [0x0006b] in <29fa7683355b471a9cc202367184a830>:0 
  at CKAN.NetFileCache.ZipValid (System.String filename, System.String& invalidReason) [0x00003] in <29fa7683355b471a9cc202367184a830>:0 
  at CKAN.NetModuleCache.Store (CKAN.CkanModule module, System.String path, System.String description, System.Boolean move) [0x0006c] in <29fa7683355b471a9cc202367184a830>:0 
  at CKAN.NetAsyncModulesDownloader.ModuleDownloadComplete (System.Uri url, System.String filename, System.Exception error) [0x00048] in <29fa7683355b471a9cc202367184a830>:0 
  at CKAN.NetAsyncDownloader.FileDownloadComplete (System.Int32 index, System.Exception error) [0x0026f] in <29fa7683355b471a9cc202367184a830>:0 
  at CKAN.NetAsyncDownloader+<>c__DisplayClass16_0.<DownloadModule>b__1 (System.Object sender, System.ComponentModel.AsyncCompletedEventArgs args) [0x00012] in <29fa7683355b471a9cc202367184a830>:0 
  at CKAN.NetAsyncDownloader+NetAsyncDownloaderDownloadPart.<ResetAgent>b__20_1 (System.Object sender, System.ComponentModel.AsyncCompletedEventArgs args) [0x00008] in <29fa7683355b471a9cc202367184a830>:0 
  at System.Net.WebClient.OnDownloadFileCompleted (System.ComponentModel.AsyncCompletedEventArgs e) [0x0000d] in <a85c1a570f9a4f9f9c3d2cfa5504e34f>:0 
  at System.Net.WebClient.<StartAsyncOperation>m__4 (System.Object arg) [0x00000] in <a85c1a570f9a4f9f9c3d2cfa5504e34f>:0 
  at System.Threading.QueueUserWorkItemCallback.WaitCallback_Context (System.Object state) [0x0000e] in <12b418a7818c4ca0893feeaaf67f1e7f>:0 
  at System.Threading.ExecutionContext.RunInternal (System.Threading.ExecutionContext executionContext, System.Threading.ContextCallback callback, System.Object state, System.Boolean preserveSyncCtx) [0x0008d] in <12b418a7818c4ca0893feeaaf67f1e7f>:0 
  at System.Threading.ExecutionContext.Run (System.Threading.ExecutionContext executionContext, System.Threading.ContextCallback callback, System.Object state, System.Boolean preserveSyncCtx) [0x00000] in <12b418a7818c4ca0893feeaaf67f1e7f>:0 
  at System.Threading.QueueUserWorkItemCallback.System.Threading.IThreadPoolWorkItem.ExecuteWorkItem () [0x0002a] in <12b418a7818c4ca0893feeaaf67f1e7f>:0 
  at System.Threading.ThreadPoolWorkQueue.Dispatch () [0x00096] in <12b418a7818c4ca0893feeaaf67f1e7f>:0 
  at System.Threading._ThreadPoolWaitCallback.PerformWaitCallback () [0x00000] in <12b418a7818c4ca0893feeaaf67f1e7f>:0
```

The workaround is to launch the application from terminal using command:
```
LANG=en_US.UTF-8 ckan
``` 

**Steps to reproduce**
* Set system locale to Polish (my case)
* Open CKAN
* Find (for example) Kerbal Engineer Redux
* Mark KER (or other mod) for installation
* Click 'Apply changes' button
* Click Apply
* Notice error described above, application terminates

**Comment by HebaruSan** [comment]
Looks the same as #3066, which was caused by some missing Mono packages. Unfortunately @TeddyDD did not specify which were missing, but recommend looking in that direction.

**Comment by michal2229** [comment]
> Looks the same as #3066, which was caused by some missing Mono packages. Unfortunately @TeddyDD did not specify which were missing, but recommend looking in that direction.

Looks similar indeed. I noticed @TeddyDD is also from Poland. It would seem that some locale packages are missing in my system or the code is not locale agnostic. I am not familiar with C# packages/locale settings, so I will use workaround mentioned in my description. 
Everything works properly when using this workaround, I was able to install several mods with no problem.
